bootstrap/src/lib/cache/redis.ts

```diff
@@ -1,36 +1,40 @@
-const { logger } = require('@chainlink/external-adapter')
-const { promisify } = require('util')
-const { timeout } = require('promise-timeout')
-const redis = require('redis')
+import { logger } from '@chainlink/external-adapter'
+import { promisify } from 'util'
+import { timeout } from 'promise-timeout'
+import { createClient, ClientOpts, RedisClient } from 'redis'
 
 // Connection
 const DEFAULT_CACHE_REDIS_HOST = '127.0.0.1' // IP address of the Redis server
 const DEFAULT_CACHE_REDIS_PORT = 6379 // Port of the Redis server
-const DEFAULT_CACHE_REDIS_PATH = null // The UNIX socket string of the Redis server
-const DEFAULT_CACHE_REDIS_URL = null // The URL of the Redis server
+const DEFAULT_CACHE_REDIS_PATH = undefined // The UNIX socket string of the Redis server
+const DEFAULT_CACHE_REDIS_URL = undefined // The URL of the Redis server
```

Comment:
Please revert this just in case. Redis client lib is especially picky and had different behavior if you feed it `null` vs `undefined` 😒
```suggestion
const DEFAULT_CACHE_REDIS_PATH = null // The UNIX socket string of the Redis server
const DEFAULT_CACHE_REDIS_URL = null // The URL of the Redis server
```

Reply:
Will revert, the official Redis types are looking for `undefined` so I will add a band-aid. Weird that it would differ :confused: